### DISCUSSION TITLE
My Student Package

### DISCUSSION THREAD

**Discussion by birdsannoy** [community/community#59099] [GitHub Education]
### Select Topic Area

Bug

### Body

Its been roughly a year me trying to apply for the sutdent pack. its been 10 times. applying. i gave them diffrent proofes. of school emails. and they rigected it. i gave them school portal and sensitive details like my student code, name and address and got refused by a bot and got redirected to most common problems. i gave them link to the portal and my code that they can login and view my acadimic status. i also gave them a pic from the portal about my acadimic status. like idk what to do anymore. should i just facetime them now and make a tour in my school or what... im loosing hope in this pack

**Comment by leonden** [comment]
For me it was as easy as logging into GitHub with my school email. Either this or registering an account with my Education email. The student/education pack was then activated automatically.
Hope this helps 😄

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @birdsannoy , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by steferd6** [comment]
I can totally understand your frustration — sometimes the verification bots make the process harder than it should be. In my case, I had a similar experience with another platform when applying for discounts for my [Abu Dhabi City Tour](https://www.mynexttriptourism.com/tour?location_id=13) project as a student, and what finally worked was contacting support with a very short, clear explanation + only one official document (not multiple). Maybe try simplifying your submission instead of giving too much info at once — that often avoids auto-flags.
